### ISSUE TITLE
Use Full Error API

### ISSUE THREAD

**Issue by alexisvincent** [systemjs/systemjs#1424]
@guybedford It would seem that when an error is encountered in imported source, an Error object is created, with stack trace as part of the error.message, any chance the stack tract could be instead set on error.stack? 

I'm not 100% sure the issue is here. But Im trying to do the following. 

When hot reloading src files, sometimes there are errors in the code. For instance, 

```js
Error: (SystemJS) Cannot read property 'lol' of undefined
	TypeError: Cannot read property 'lol' of undefined
	    at DashboardContainer.render (http://localhost:3000/app/routes/dashboard/dashboard.js!transpiled:51:20)
	    at DashboardContainer.render (http://localhost:3000/build/dependencies.js:27448:32)
	    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext 
    ...
```

I want to catch these and then display then in a component, so that developers don't need to check the console. But rather get the message full screen. 

Im using redbox-react for this, which supports the Error api... However it would seem that the errors generated are constructed fully via the first arg of Error, namely the message attribute.

Any chance this could be changed to better follow the Error api, or is the issue not here?

**Comment by guybedford** [comment]
Sure, we can assign the stack as well as the message.

**Comment by guybedford** [comment]
Added https://github.com/systemjs/systemjs/commit/2d1c8c935fe98036beebb5144f7c77a2451f0695.

**Comment by alexisvincent** [comment]
Great. I did some reading, would seem this is more of a defacto standard then a formal one. But still nice to support

**Comment by guybedford** [comment]
Released in 0.19.38.
